datacite/client.py

```diff
@@ -77,12 +60,12 @@ def doi_get(self, doi):
 
         :param doi: DOI name of the resource.
         """
-        r = self._request_factory()
-        r.get("doi/" + doi)
-        if r.code == 200:
-            return r.data
+        request = self._create_request()
+        resp = request.get("doi/" + doi)
+        if resp.status_code == 200:
```

Comment:
nice-to-have: Could we sould constants for this codes? e.g. [Werkzeug](https://github.com/pallets/werkzeug/blob/c4d85cb9e0e78c2d5786baad649100d42526c399/src/werkzeug/http.py#L112) ones (I'm not sure if these are the ones you mentioned IRL Nico). This applies in general for Invenio, might be a good thing to use more "human readable" consts (even if we still should know what a 200, 201, 404 etc. are)